bindings/python/_broker.cpp

```diff
@@ -169,134 +173,168 @@ PYBIND11_MODULE(_broker, m) {
     .def("capacity", &broker::publisher::capacity)
     .def("fd", &broker::publisher::fd)
     .def("drop_all_on_destruction", &broker::publisher::drop_all_on_destruction)
-    .def("publish", (void (broker::publisher::*)(broker::data d)) &broker::publisher::publish)
-    .def("publish_batch",
-       [](broker::publisher& p, std::vector<broker::data> xs) { p.publish(xs); })
+    .def("publish", (void(broker::publisher::*)(broker::data d))
+                      & broker::publisher::publish)
```

Comment:
I'm not sure why this is separating the `&` from the variable name, if the intention is for it to be taking the address of that variable.

Reply:
I'm not sure either was `clang-format` is thinking about this code. Maybe it confuses this with an argument declaration. Although, then it should put the `&` after the brace... 🤷🏻‍♂️